docs/application/native/guides/app-management/ui-app/dali-widget-app.md

```diff
@@ -0,0 +1,268 @@
+|--------------------|-----------------------------------------------------------------------------------------------|
+| `OnCreate()`       | This function is called after the widget instance is created.                                 |
+| `OnTerminate()`    | This function is called after the widget instance is terminated.                              |
+| `OnPause()`        | This function is called when the widget is invisible.                                         |
+| `OnResume()`       | This function is called when the widget is visible.                                           |
+| `OnResize()`       | This function is called before the widget size is changed.                                    |
+| `OnUpdate()`       | This function is called when an event for updating the widget is received.                    |
+| `SetContentInfo()` | This function sends data about current state of widget instance to widget viewer application. |
+
+Most of functions are used for managing widget instance lifecycle. You can describe what will be done on each lifecycle functions.
+
+Widget instance can send data to widget viewer application using `SetContentInfo()` function. If you want to save the current state of widget before deleting it, view the current state of the widget and delete it.
+
+<a name="create"></a>
+## The steps to create a DALi widget application:
+
+1. Initialize the widget application:
+
+   To create the `Dali::WidgetApplication` class instance, initialize and set up DALi for a widget application:
+   ```
+   int DALI_EXPORT_API main( int argc, char **argv )
+   {
+     WidgetApplication application = WidgetApplication::New( &argc, &argv, "" );
+
+     WidgetApplicationExample test( application );
+
+     application.MainLoop();
+
+     return 0;
+   }
+   ```
+
+2. Connect signals to keep yourself informed when certain system events occur.
+
+   To manage signal connection safely, DALi provides the `Dali::ConnectionTracker` class. A typical way to start a DALi application is to create a class derived from the `Dali::ConnectionTracker` class and use its member functions as callback functions for DALi signals. When the application receives the `InitSignal()` function, it can create user interface components. Additionally, connect the callback `WidgetApplicationExample::Create()` function to the `Dali::WidgetApplication::InitSignal()` function:
+   ```
+    class WidgetApplicationExample : public ConnectionTracker
+    {
+      public:
+        WidgetApplicationExample( WidgetApplication& application )
+          : mApplication( application )
+        {
+          mApplication.InitSignal().Connect( this, &WidgetApplicationExample::Create );
+        }
+
+        void Create( Application& application )
+        {
+          // Register widget creating function here.
+        }
+
+      private:
+        WidgetApplication& mApplication;
+    }
+   ```
+
+3. Create own widget class
```

Comment:
Please can you change the sentence as:
Create a widget class.

Reply:
Done